### ISSUE TITLE
Accuracy, Loss and Perplexity

### ISSUE THREAD

**Issue by helderduartemop** [tensorflow/magenta#1723]
I've been doing research with Polyphony_RNN to try to generate my own music from my dataset, but I still don't understand what is defined by accuracy, loss and perplexity.
Is accuracy the predicted notes from the model vs the original set? (So that the next notes in the training set also work as a validation set?)

Is loss simply the cross-entropy of the seq2seq model?

By reading the code, I figured that perplexity is simply the exponential of the loss, but am I missing something?

Thanks in advance.
